src/components/Menu/ControlledMenu.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import omit from 'lodash/omit';
+import { getUniqueHash } from '../../utils';
+import { Button, ButtonIcon } from '../../';
+
+const propTypes = {
+  /**
+   * The button that triggers the dropdown menu
+   * ```
+   * {
+   *    icon: 'settings',
+   *    sprite: 'utility',
+   *    title: 'Click me',
+   *    noBorder: true,
+   * }
+   * ```
+   */
+  button: PropTypes.shape({
```

Comment:
At this point and with #291 landing, it would make more sense to always pass a button imo.

Reply:
Okay